src/main/java/com/vilgodskaia/movieplatformpetproject/config/aspect/MovieAop.java

```diff
@@ -0,0 +1,74 @@
+package com.vilgodskaia.movieplatformpetproject.config.aspect;
+
+import com.vilgodskaia.movieplatformpetproject.util.DataType;
+import lombok.extern.log4j.Log4j2;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+
+@Component
+@Aspect
+@Log4j2
+public class MovieAop {
+
+    private final String apiPointCutMovieController = "execution(* com.vilgodskaia.movieplatformpetproject.api.movie.MovieRestController.**(..))";
+    private final String apiPointCutStreamingController = "execution(* com.vilgodskaia.movieplatformpetproject.api.streamingplatform.StreamingPlatformRestController.**(..))";
+    private final String apiPointCutMovieOnStreamingController = "execution(* com.vilgodskaia.movieplatformpetproject.api.movieonstreamingplatform.MovieOnStreamingPlatformRestController.**(..))";
+
+    private final String exceptionPointCut = "execution(* com.vilgodskaia.movieplatformpetproject.*.*.*(..))";
+
+    @Pointcut(apiPointCutMovieController)
+    public void logMovieController() {
+    }
+
+    @Pointcut(apiPointCutStreamingController)
+    public void logStreamingController() {
+    }
+
+    @Pointcut(apiPointCutMovieOnStreamingController)
+    public void logMovieOnStreamingController() {
+    }
+
+    @Pointcut("logMovieController() || logStreamingController() || logMovieOnStreamingController()")
+    public void logAnyController() {
+    }
+
+    @Before("logAnyController()")
+    public void logRequest(JoinPoint joinPoint) {
+        log.info("METHOD: " + joinPoint.getSignature().getName());
+        log.info(createJoinPointForLogs(joinPoint, DataType.REQUEST));
+    }
+
+    @AfterReturning("logAnyController()")
```

Comment:
https://stackoverflow.com/questions/18469703/log-the-return-value-of-a-method-using-spring-aop